src/main/webapp/i18n/de/student-dashboard.json

```
@@ -46,7 +46,13 @@
                 "noUpcomingSession": "Kein Termin"
             },
             "menu": {
-                "exercises": "Aufgaben",
+                "exercise": "Aufgabe",
```

There seems to be a typo in the translation key 

```suggestion
                "exercises": "Aufgabe",
```

---

Good Catch!